test/support/generator/random-value-generator.ts

```diff
@@ -8,8 +8,13 @@ export class RandomValueGenerator {
     this.chance = new Chance(RandomValueGenerator.seed);
   }
 
-  string(): string {
-    return this.chance.string();
+  string({ minLength, maxLength }: { minLength?: number; maxLength?: number } = {}): string {
+    if (minLength || maxLength) {
+      const length = this.chance.integer({ min: minLength ?? 0, max: maxLength ?? Math.max(20, minLength * 2) });
+      return this.chance.string({ length: length });
+    } else {
+      return this.chance.string();
+    }
   }
```

Comment:
This has a slight bug if either of `minLength` or `maxLength` are 0, since 0 is falsey.
You could do
```
string({ minLength, maxLength }: { minLength?: number; maxLength?: number }): string {
  const minLengthToUse = minLength ?? 0;
  const maxLengthToUse = maxLength ?? Math.max(20, minLengthToUse * 2);

  const length = this.chance.integer({ min: minLengthToUse, max: maxLengthToUse, });
  return this.chance.string({ length });
}
```

Reply:
Good spot, changed